lib/commands/context.js

```diff
@@ -51,13 +45,24 @@ commands.setContext = async function setContext (name) {
     return;
   }
 
-  let contexts = await this.getContexts();
+  let webviewsMapping = [];
+  if (!this.isChromeSession) {
+    // if we don't have a Chrome browser session, we use ADB to figure out which webviews are available
+    const opts = {
+      androidDeviceSocket: this.opts.androidDeviceSocket,
+      ensureWebviewsHavePages: this.opts.ensureWebviewsHavePages,
+      webviewDevtoolsPort: this.opts.webviewDevtoolsPort,
+      enableWebviewDetailsCollection: true
```

Comment:
In order to get [info object](https://github.com/appium/appium-android-driver/blob/master/lib/commands/context.js#L81-L89) and keep performance, it breaks backward compatibility by specifying `enableWebviewDetailsCollection: true`. Let me know if you have a concern.

Reply:
I'm fine to change it to true by default. Although, explicit setting it to false must be properly respected and the test should not fail in case this info cannot be fetched

Reply:
Sounds good. Thank you for your suggestion. Addressed in a83c7b2 and https://github.com/appium/appium-android-driver/pull/699/commits/8cd5414555d626457760599550168e7f30dc170c